core/container_exec.go

```diff
@@ -57,6 +57,9 @@ type ContainerExecOpts struct {
 	// Redirect the command's standard error to a file in the container
 	RedirectStderr string `default:""`
 
+	// Combine the command's standard output and standard error to the standard output
+	CombinedOutput bool `default:"false"`
```

Comment:
suggestion: if we keep this behaviour, indicates it's `stderr` redirected to `stdout` and so combined output is available under `stdout` but not under `stderr`


```suggestion
	RedirectStderrToStdout bool `default:"false"`
```